versions/v5/functions/display-logic.php

```diff
@@ -0,0 +1,231 @@
+		$description = wptexturize( strip_tags( $story_description, '<b><em><i><u><strong>' ) );
+	}
+	elseif ( $story_subtitle = get_post_meta( $story->ID, 'story_subtitle', true ) ) {
+		$description = wptexturize( strip_tags( $story_subtitle, '<b><em><i><u><strong>' ) );
+	}
+
+	$vertical = null;
+	if ( $show_vertical ) {
+		$vertical = get_the_category( $story->ID );
+		if ( $vertical ) {
+			$vertical = $vertical[0];
+			$vertical = wptexturize( $vertical->name );
+		}
+	}
+
+	ob_start();
+?>
+<article class="fp-feature <?php echo $css_class; ?>">
+	<a class="fp-feature-link" href="<?php echo get_permalink( $story->ID ); ?>">
+		<?php if ( $thumbnail ): ?>
+		<div class="fp-feature-img-wrap">
+			<?php echo $thumbnail; ?>
+
+			<?php if ( $show_vertical && $vertical ): ?>
+			<span class="fp-vertical">
+				<?php echo $vertical; ?>
+			</span>
+			<?php endif; ?>
+		</div>
+		<?php endif; ?>
+	</a>
+	<div class="fp-feature-text-wrap">
+		<<?php echo $heading; ?> class="fp-feature-title">
+			<a class="fp-feature-link" href="<?php echo get_permalink( $story->ID ); ?>">
+				<?php echo $title; ?>
+			</a>
+		</<?php echo $heading; ?>>
+		<div class="fp-feature-description">
+			<?php echo $description; ?>
+		</div>
+	</div>
+</article>
+<?php 	return ob_get_clean();
+}
+
+
+/**
+ * Displays a single Today article on the front page.
+ **/
+function display_front_page_today_story( $article ) {
+	$url = $article->get_link();
+	$title = $article->get_title();
+	$publish_date = $article->get_date('m/d');
+
+	ob_start();
+?>
+<article class="fp-today-feed-item">
+	<a class="fp-today-item-link" href="<?php echo $url; ?>">
+		<div class="publish-date"><?php echo $publish_date; ?></div>
+		<?php echo $title; ?>
+	</a>
+</article>
+<?php 	return ob_get_clean();
+}
+
+
+/**
+ * Displays a single event item on the front page.
+ **/
+function display_front_page_event( $event ) {
+	$start = strtotime( $event['starts'] );
+	$description = substr( strip_tags( $event['description'] ), 0, 250 );
+	if ( strlen( $description ) == 250 ) {
+		$description .= '...';
+	}
+
+	ob_start();
+?>
+<div class="fp-event">
+	<div class="fp-event-when">
+		<span class="fp-event-day"><?php echo date( 'D', $start ); ?></span>
+		<span class="fp-event-date"><?php echo date( 'd', $start ); ?></span>
+		<span class="fp-event-month"><?php echo date( 'M', $start ); ?></span>
+	</div>
+	<div class="fp-event-content">
+		<span class="fp-vertical"><?php echo $event['category']; ?></span>
+		<span class="fp-event-title">
+			<a class="fp-event-link" href="<?php echo $event['url']; ?>"><?php echo $event['title']; ?></a>
+		</span>
+		<div class="fp-event-description">
+			<?php echo $description; ?>
+		</div>
+	</div>
+</div>
+<?php 	return ob_get_clean();
+}
+
+
+/**
+ * Displays a single featured gallery on the front page.
+ **/
+function display_front_page_gallery( $gallery, $css_class='' ) {
+	if ( !$gallery ) { return false; }
```

Comment:
Curly brackets can be removed.

Reply:
Updated in both `display-logic.php` files.